src/current/_includes/releases/v23.2/v23.2.0-alpha.7.md

```diff
@@ -0,0 +1,56 @@
+## v23.2.0-alpha.7
+
+Release Date: November 20, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-0-alpha-7-sql-language-changes">SQL language changes</h3>
+
+- Previously, if [session variable `use_declarative_schema_changer`](../v23.2/session-variables.html#use_declarative_schema_changer) was set to `off`, then [`ALTER TABLE ... ALTER COLUMN ... SET NOT NULL`](../v23.2/alter-table.html#alter-column) was run on a column which contained a NULL value, an error with code `23514` (`check_violation`) would be returned. Now in this scenario the error returned will have code 23502 (`not_null_violation`) to match [PostgreSQL](https://www.postgresql.org/docs/8.4/errcodes-appendix.html). [#113970][#113970]
+- The `sql.txn.read_committed_syntax.enabled` cluster setting was renamed to [`sql.txn.read_committed_isolation.enabled`](). [#113833][#113833]
```

Comment:
first use of the term "cluster setting" could be a link to https://www.cockroachlabs.com/docs/v23.2/cluster-settings

the second cluster setting name is a link but it's broken b/c it points to nowhere

Reply:
added correct links.